book/src/pong/09_keeping_score.md

```diff
@@ -117,7 +117,7 @@ Priority `P1` is a sensible option for these objects, so we'll do that for the b
 ```rust
 Object::new(sprites::BALL.sprite(0))
     .set_pos(self.pos.round())
-    .set_priority(Priority::P1)
```

Comment:
We've used it as `Priority` in the above cases, so it feels like this should've been imported already?